eth/tracers/tracer.go

```diff
@@ -335,6 +336,9 @@ func New(code string) (*Tracer, error) {
 		depthValue:      new(uint),
 		refundValue:     new(uint),
 	}
+	tracer.ctx["gasPrice"] = vmctx.GasPrice
+	tracer.ctx["txGas"] = params.TxGas
```

Comment:
Hardcoded 21K? 
You should at least put a TODO here so it doesn't get forgotten, in case you plan to fix this in a later PR

Reply:
I wanted to see if we can avoid hardcoding a value here. It seems `Tracer.CaptureEnd` is already exposing the gas usage to JS. However the issue is this gas usage doesn't include the intrinsic cost. That's because the tracer is invoked within `EVM.Call` or `EVM.Create`, but intrinsic gas is already reduced before in `StateTransition.TransitionDb`. How do you think it's best to proceed from here?

Reply:
Well, one way might be to expose `core.IntrinsicGas` so it's callable from the js.